src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -28,47 +38,94 @@ export const mhvBaseUrl = () => {
+    case frontendApps.RX:
+      return 'Prescription Refill and Tracking';
 
-    case frontendApps.LETTERS:
-      return '/records/download-va-letters/letters';
+    case frontendApps.MESSAGING:
+      return 'Secure Messaging';
 
-    case frontendApps.VETERAN_ID_CARD:
-      return 'https://vicbdc.ppd.vba.va.gov/VIC';
+    case frontendApps.LAB_AND_TEST_RESULTS:
+      return 'Lab and Test Results';
 
-    default:
-      return '/';
+    case frontendApps.APPOINTMENTS:
+      return 'VA Appointments';
+
+    default: // Not a recognized health tool.
   }
+
+  return null;
 };
 
-export const redirectUrl = contentUrl => {
-  switch (contentUrl) {
-    case '/health-care/secure-messaging/':
-      return `${mhvBaseUrl()}/mhv-portal-web/secure-messaging`;
+export const toolUrl = appId => {
+  switch (appId) {
+    case frontendApps.HEALTH_RECORDS:
+      return {
+        url: `${mhvBaseUrl()}/mhv-portal-web/download-my-data`,
+        redirect: false,
+      };
+
+    case frontendApps.RX:
+      return {
+        url: `${mhvBaseUrl()}/mhv-portal-web/web/myhealthevet/refill-prescriptions`,
+        redirect: true,
+      };
 
-    case '/health-care/refill-track-prescriptions/':
-      return `${mhvBaseUrl()}/mhv-portal-web/web/myhealthevet/refill-prescriptions`;
+    case frontendApps.MESSAGING:
+      return {
+        url: `${mhvBaseUrl()}/mhv-portal-web/secure-messaging`,
+        redirect: true,
+      };
 
-    case '/health-care/get-medical-records/':
-      return `${mhvBaseUrl()}/mhv-portal-web/download-my-data`;
+    case frontendApps.APPOINTMENTS:
+      return {
+        url: `${mhvBaseUrl()}/mhv-portal-web/web/myhealthevet/scheduling-a-va-appointment`,
+        redirect: true,
+      };
 
-    case '/health-care/schedule-view-va-appointments/':
-      return `${mhvBaseUrl()}/mhv-portal-web/web/myhealthevet/scheduling-a-va-appointment`;
+    case frontendApps.LAB_AND_TEST_RESULTS:
+      return {
+        url: `${mhvBaseUrl()}/mhv-portal-web/labs-tests`,
+        redirect: true,
+      };
 
-    case '/health-care/view-test-and-lab-results/':
-      return `${mhvBaseUrl()}/mhv-portal-web/labs-tests`;
+    case frontendApps.CLAIMS_AND_APPEALS:
+      return {
+        url: '/track-claims/',
+        redirect: true,
+      };
 
-    case '/claim-or-appeal-status/':
-      return '/track-claims/';
+    case frontendApps.GI_BILL_BENEFITS:
+      return {
+        url: '/education/gi-bill/post-9-11/ch-33-benefit/status',
+        redirect: false,
+      };
+
+    case frontendApps.DISABILITY_BENEFITS:
+      return {
+        url:
+          '/disability-benefits/apply/form-526-disability-claim/veteran-information',
+        redirect: false,
+      };
+
+    case frontendApps.LETTERS:
+      return {
+        url: '/records/download-va-letters/letters',
+        redirect: false,
+      };
+
+    case frontendApps.VETERAN_ID_CARD:
+      return {
+        url: 'https://vicbdc.ppd.vba.va.gov/VIC',
+        redirect: false,
+      };
```

Comment:
It might help to show the relationship between values by creating a config map similar to [this](https://github.com/department-of-veterans-affairs/vets-website/blob/master/script/constants/hostnames.js), but that's probably just a preference.

Reply:
I think an Object/Map would be more readable. I was thinking that a switch would use less memory, but that's probably moot given the small number of cases/members.